beta/plugins/markdownToHtml.mjs

```diff
@@ -6,18 +14,16 @@ const unrwapImages = require('remark-unwrap-images'); // Removes <p> wrapper aro
 const smartyPants = require('./remark-smartypants'); // Cleans up typography
 const html = require('remark-html');
 
-module.exports = {
-  remarkPlugins: [
-    externalLinks,
-    customHeaders,
-    images,
-    unrwapImages,
-    smartyPants,
-  ],
-  markdownToHtml,
-};
+export const remarkPlugins = [
+  remarkGfm,
+  externalLinks,
+  customHeaders,
+  images,
+  unrwapImages,
+  smartyPants,
+];
 
-async function markdownToHtml(markdown) {
+export async function markdownToHtml(markdown) {
   const result = await remark()
```

Comment:
What is this file doing? How is this function used?
Is the input MDX or markdown?
If it’s MDX, then there will be JSX components, right? Shouldn’t those components be evaluated?

Reply:
A recent version of this pipeline would be:

```js
import {unified} from 'unified'
import remarkParse from 'remark-parse'
import remarkImages from 'remark-images'
import remarkUnwrapImages from 'remark-unwrap-images'
import remarkSmartyPants from './your-custom-plugin.js'
import remarkRehype from 'remark-rehype'
import rehypeAutolinkHeading from 'rehype-autolink-headings'
import rehypeExternalLinks from 'rehype-external-links'
import rehypeStringify from 'rehype-stringify'

const result = await unified()
  .use(remarkParse) // Parse markdown
  .use(remarkImages) // Simpler image syntax (no `alt` support though!! Do you need this?)
  .use(remarkUnwrapImages) // Remove `<p>` wrapper around images
  .use(remarkSmartyPants) // !!If possible I’d recommend forcing this in a style guide rather than trying to fix it like this
  .use(remarkRehype) // Turn markdown AST into HTML AST
  .use(rehypeAutolinkHeading) // Add links to headings (might want to configure to prepend an icon)
  .use(rehypeExternalLinks) // Add `rel=nofollow` and such
  .use(rehypeStringify) // Serialize HTML
  .process('string')
  ```
  
But the big question remains is whether you’re dealing with ESM, expressions, and JSX that needs to be evaluated or not?

Reply:
I think you can pretty much ignore this function, it's only used for some tangential thing (i think maybe on the blog) which isn't hooked up to the beta site yet